rotkehlchen/tasks/assets.py

```diff
@@ -0,0 +1,42 @@
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.assets.utils import check_if_spam_token
+from rotkehlchen.db.dbhandler import DBHandler
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.types import SPAM_PROTOCOL
+
+SYMBOL_AND_NAME_ASSETS_QUERY = (
+    'SELECT C.symbol, A.name, A.identifier FROM evm_tokens as B LEFT JOIN '
+    'common_asset_details AS C ON C.identifier = B.identifier JOIN assets as A on '
+    'A.identifier=B.identifier WHERE B.PROTOCOL IS NOT ? ORDER BY C.symbol'
+)
+
+
+def autodetect_spam_assets_in_db(user_db: DBHandler) -> None:
+    """Autodetect spam tokens already in the globaldb and ignore them"""
+    globaldb = GlobalDBHandler()
+    detected_spam_assets = []
+
+    # get the spam tokens already in the globaldb
```

Comment:
This comment really confused me

```suggestion
    # Check if any tokens in the global DB not marked as spam are actually matching the spam patterns
```